build.sh

```diff
@@ -1,14 +1,13 @@
 #!/bin/bash
-
 dotnet restore
-
-cd src/Serilog/
-
-dotnet build -f netstandard1.0 -c Release
-dotnet build -f netstandard1.3 -c Release
-
-cd ../..
-cd test/Serilog.Tests/ 
-
-dotnet build -f netcoreapp1.0 -c Release
-dotnet test -f netcoreapp1.0  -c Release
\ No newline at end of file
+for path in src/*/project.json; do
+    dirname="$(dirname "${path}")"	
+    dotnet build ${dirname} -f netstandard1.0 -c Release
+    dotnet build ${dirname} -f netstandard1.3 -c Release
```

Comment:
@khellang not a stupid question at all, I was having all sorts of issues with the targets not picking up the framework in `project.json` on nix based machines.

When not setting the arg, I was getting `error DOTNET1011: Framework not installed: .NETFramework,Version=v4.5.2 in`

Could be my tooling tho.

```
.NET Command Line Tools (1.0.0-preview1-002702)

Product Information:
 Version:     1.0.0-preview1-002702
 Commit Sha:  6cde21225e

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.11
 OS Platform: Darwin
 RID:         osx.10.11-x64
```  
I will remove the targets on the lib however not sure we can on the tests ATM.